politeiawww/config.go

```diff
@@ -125,6 +125,10 @@ type config struct {
 	VoteDurationMax          uint32 `long:"votedurationmax" description:"Maximum duration of a proposal vote in blocks"`
 	AdminLogFile             string `long:"adminlogfile" description:"admin log filename (Default: admin.log)"`
 	Mode                     string `long:"mode" description:"Mode www runs as. Supported values: piwww, cmswww"`
+	CmsHost                  string `long:"cmshost" description:"Cms ip:port"`
```

Comment:
Would prefer initialism be consistent case.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms